pkg/starboard/config.go

```diff
@@ -364,6 +365,35 @@ func (c ConfigData) GetTrivyMode() (TrivyMode, error) {
 		value, keyTrivyMode, Standalone, ClientServer)
 }
 
+func (c ConfigData) GetScanJobAnnotations(namespace string) (map[string]string, error) {
+	var customAnnotationsKey string
+	switch namespace {
+	case NamespaceName:
+		customAnnotationsKey = AnnotationCustomAnnotationsForScanJobPods
+	case starboardOperator.NamespaceName:
+		customAnnotationsKey = starboardOperator.AnnotationCustomAnnotationsForScanJobPods
```

Comment:
Define a new param called `scanJob.annotations`. Similar to what we did for `scanJob.tolerations`. There's no need to prefix a key for CLI and operator. They have their own copy of the ConfigMap. So just read key from the map